### PR TITLE
Welcoming contact field

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by n0toose** [comment]
Why do you exactly want to remove the parts of a document that say "Be kind, respectful, constructive, accountable (and learn from it), act in the best interest of the community and *do not* harass, troll, dox or be a pervert", as well as treating people for what they are, do and say, and not what they do, while implicitly making the maintainer here a BDFL, in the name of being "pro-community"?

I don't think I've ever seen an answer that holds in a meritocratic manner other than "it can be abused", as if it were a legally binding document and not a means of providing direction, as well as personal hunches against the original author of the document, but I promise that I am asking in good faith and out of curiosity. The only thing I would personally find myself very skeptical about is the concept of permanence expressed in it (see: life imprisonment, yay or nay?) and most of the cons that I have seen are things that could happen just as easily without that document, but that's just me.

**Comment by GeopJr** [comment]
Thanks for your contribution!

Unfortunately, there are no plans on removing the Code of Conduct. This is not open for debate either.

I understand that you might no longer feel comfortable with your translation existing in the project, so I'll leave you sometime to decide before merging the Weblate PR.

As for the Glossary issue, thanks for bringing it to my attention, I'm still getting used to Weblate (just migrated to it yesterday 😅)!

**Comment by comradekingu** [comment]
@n0toose I appreciate your inquisitive angle. If that is all of what it said, no problem. You also added stuff that isn't there.
Those words are just padding. It amounts to being rosy language embedded in functionality that does something entirely different.
To compare it against a proposed removal, some things happen to be retained, but more importantly what you didn't mention is lost.
[summary]"Here" is only a ephemeral question of what the community wants to be or ultimately fork, rather than other provisions dictating it to be. A never perfect onset doesn't legitimize actions without further analysis, ala. the forever onside "we tried to do something" dwindling into more and more mutually exclusive logic.
[/summary]

The idea that anti-social behavior enough to warrant mention is prevented by having it available in writing amounts to unwritten rules in writing. As such it is redundant to law. It is also redundant to the BDFL-concept, except this is purposefully undermined.
You will find "project maintainers" replaced with "community leaders". The expressed goal is to undermine meritocracy.

What you then do is transfer power. Away from people with vested interest in creating something, onto people that can gain power without it.
That is its functional use removed from its omission. When it is used as such, that is its use.
It comes as no surprise the attraction garners people of anti-social proclivity to do so abusively, with many examples to draw from. This is the gain of function otherwise made much more difficult.

Something that escalates conflict beyond the control of involved parties, in required secrecy, with no checks and balances isn't community any more than its "Other conduct which could reasonably be considered inappropriate in a professional setting" being _unacceptable_. Those are corporate values for corporate ends, not community.

To make an further example of how authoritarian it is, consider the writing and reading of "harassment-free" a concept entirely within its making.
It sounds good as a concept, but "harassment" is removed from its legal meaning, and taken to mean objectivity doesn't exist.
There can be no agreed upon standard in which "sexual attention or advances of any kind" doesn't necessitate negotiation of social hierarchies. Sexuality in itself requires an advance. No society has ever been successful in undoing biology.
Examples from the originator of CoCs, the Catholic church onwards are not examples to follow, and certainly not made in an aim to strengthen community. It says "healthy community", but are any of those anything but unhealthy?

To further put the code above all else, and penalize with ultimate and permanent authority is not even as absurd as retroactively removing all traces. That is a socio-political system, in a document that outright bans "political attacks".

Thinking there is any hiding from it is a futile endeavour.
"This includes avoiding interactions in community spaces as well as external channels like social media."
There is plenty more that is just wrong, badly written, or downright asinine, but it is more water under a fallen bridge.

@GeopJr
Countless of these have been removed, as they do not hold more value than that.
They are never (?) instated by asking the community, and allegiance to it is just implied.
It isn't mentioned in the Hosted Weblate project.
I stopped short of contributing because I am only in it to spread freedom.

TL;DR Is there any benefit to be had in the former, over what amounts to a contact field with an appeal to constructive reasoning? Haven't yet seen an answer outside of "This is not open for debate" "green lights" "important" or "a guide said so".

**Comment by Yimura** [comment]
image

**Comment by n0toose** [comment]
> Something that escalates conflict beyond the control of involved parties, in required secrecy, with no checks and balances isn't community any more than its "Other conduct which could reasonably be considered inappropriate in a professional setting" being unacceptable. Those are corporate values for corporate ends, not community.

That was insightful, thanks.

> I appreciate your inquisitive angle. If that is all of what it said, no problem. You also added stuff that isn't there.

I went through each bullet point precisely and then summed it up with one word in "Our Standards", in the order of appearance, minus the "don't be a pervert" because it wouldn't syntactically make sense, in order to specifically prevent you from accusing me of misrepresenting its contents and immediately shoot it down. For more clarity, I briefly mentioned the Enforcement Guidelines section and skipped "Scope", "Enforcement", "Attribution" and "Our Pledge", and then I think that I believed that you went on to hijack what I said in order to analyze your ideological interpretation of the entire document without directly connecting it with any of the things that I said whatsoever. I will not comment on any of the pseudo-Freudian "the lines between sexual advances and harassment are very vague" and "sex, human nature, etc." stuff, because I don't think that any of us should have to play the role of a mother or a sex therapist. The general rule of thumb is "Use your brain, cut it off when needed, a failed flirt isn't the end of the world", it's whatever, miscommunications can happen, unless if you open up with sending NSFW pictures to people, which is more than obvious. We aren't here to create some "Ivory Tower" specifically meant for shooting at people with bows and arrows.

> It sounds good as a concept, but "harassment" is removed from its legal meaning, and taken to mean objectivity doesn't exist.

I believe that you are aware that the legal term "harassment" changes from jurisdiction to jurisdiction (Conclusion, it doesn't exist), because not all countries have the same laws or legal definitions, which is why it was explicitly defined in literally the first paragraph.

> What you then do is transfer power. Away from people with vested interest in creating something, onto people that can gain power without it.

-v, and, yet again, literally who cares and what sort of power does one person even supposedly get? Deleting issues on GitHub.com and bragging rights? Maybe a conference talk? What is the criteria for a person not having an interest in creating something? Do you even fathom how much of a microcosm this entire community online is?

I can't even blame @Yimura for pasting a picture of grass in reference to the phrase "touch grass", because it feels like the significance of what's going on here has been amplified greatly.

**Comment by GeopJr** [comment]
As per my original comment, this is not up for debate.

It's fine if you don't feel comfortable contributing due to the existence of the CoC. I haven't merged your translation yet, giving you the opportunity to remove it if you no longer wish for it to be merged.

I value n0toose's position on this as they had to deal with similar situations at Tenacity.

As mentioned before, I migrated to Weblate last night, I haven't had the chance to go over everything. If you believe adding "This porject has a Code of Conduct" somewhere there is needed, I'll do so.

Lastly, for future readers, please do not open an issue or pr demanding the removal of the CoC. It's here to stay.

P.S. Yimura and Jonohas are not contributors.
